src/Symfony/Component/Console/Tester/CommandTester.php

```diff
@@ -24,9 +24,12 @@ class CommandTester
 {
     use TesterTrait;
 
+    private Command $command;
+
     public function __construct(
-        private Command $command,
+        object $command,
     ) {
+        $this->command = Command::createFromAttribute($command);
```

Comment:
The static method can be skipped and the logic to instantiate the command inlined in this constructor. You can copy the [code from `Application::addCommand`](https://github.com/symfony/symfony/blob/50d7b25eeccd273042ea46137b52b36ff710d72b/src/Symfony/Component/Console/Application.php#L554-L571).